tests/jsx_escape/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -42,9 +42,9 @@ raw_amp = <span>foo & bar</span>
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 many_nbsp = <div>&nbsp; &nbsp; </div>;
 single_nbsp = <div>&nbsp;</div>;
-many_raw_nbsp = <div>&nbsp;&nbsp;&nbsp;</div>;
+many_raw_nbsp = <div> </div>;
 
-amp = <span>foo & bar</span>;
+amp = <span>foo &amp; bar</span>;
```

Comment:
😄 you're totally right on that point but, on the other hand, shouldn't we completely rely on the parser? cc/ @vjeux